tools/build_config/powershell_helpers.ps1

```diff
@@ -0,0 +1,15 @@
+function Write-And-Invoke([string]$command) {
```

Comment:
Since these functions are now in a shared utility file it would be worth adding some function headers: what do they do, what arguments to they take and (?) return, what system changes do they leave (e.g. change of dir).